boards/nxp/imx95_evk/imx95_evk_mimx9596_m7_ahab.yaml

```diff
@@ -0,0 +1,356 @@
+# ========================  Advanced High-Assurance Boot Configuration template for mimx9596.  =========================
+
+# ======================================================================================================================
+#                                                 == General Options ==                                                 
+# ======================================================================================================================
+# -------------------------------------===== The chip family name [Required] =====--------------------------------------
+# Description: NXP chip family identifier.
+# Possible options: <mimx8ulp, mimx9131, mimx9352, mimx9596, mimxrt1181, mimxrt1182, mimxrt1187, mimxrt1189>
+family: mimx9596
+# -----------------------------------------===== MCU revision [Optional] =====------------------------------------------
+# Description: Revision of silicon. The 'latest' name, means most current revision.
+# Possible options: <a0, a1, b0, latest>
+revision: latest
+# -----------------------------------------===== Target memory [Optional] =====-----------------------------------------
+# Description: Target memory for AHAB container
+# Possible options: <serial_downloader, standard, nand_2k, nand_4k>
+target_memory: standard
+# ---------------------------------------===== Output file name [Required] =====----------------------------------------
+# Description: Output AHAB file name
+output: ../../../build/zephyr/flash.bin
+# ---------------------------------------===== Container version [Optional] =====---------------------------------------
+# Description: Force container version, if not specified the default version is used. This configuration option is valid
+# only for chips that supports both AHAB container versions. Possible options <2, 1>
+container_version: 2
+# ------------------------------===== List of containers present in AHAB [Required] =====-------------------------------
+# Description: The order of containers in the list defines the order in AHAB.
+containers:
+  -
+  # ====================================================================================================================
+  #                                          == List of possible 2 options. ==                                          
+  #                                        Options [binary_container, container]                                        
+  # ====================================================================================================================
+
+  #  =========================== [Example of possible configuration: #0 , erase if not used] ============================
+    # -----------------------------------===== Binary AHAB container [Required] =====-----------------------------------
```

Comment:
this file cannot be here, it will be treated as a twister file

Reply:
WIll remove it and generate during build.